src/mempool.js

```diff
@@ -0,0 +1,31 @@
+const { transaction } = require("./transaction"); // Replace with actual path
+
+let mempool = null;
+
+class Mempool {
+  constructor() {
+    this.tx = [];
+  }
+
+  insert_transaction(tx) {
```

Comment:
let's use camelCase naming convention for better readability => `insertTransaction` for method name